torchao/prototype/models/gpt_fused/model.py

```diff
@@ -0,0 +1,255 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
could you add a file in test or benchmark script that would just sanity check that the script works with real and random weights

Also why prototype namespace? I think `torchao.models.gpt` is better, i expect a lot of people will use this as is

Reply:
Using prototype just to get started. Yes, we can add a benchmark script. I'll work on that next.